test-it/karma/javascriptinvoke/Reinitialize.Test.js

```diff
@@ -0,0 +1,43 @@
+describe('A JavaScript function invoke', function () {
+    'use strict';
+    // Reference aliases
+    var vireoHelpers = window.vireoHelpers;
+    var vireoRunner = window.testHelpers.vireoRunner;
+    var fixtures = window.testHelpers.fixtures;
+
+    var vireo;
+
+    var jsComplexFunctionsViaUrl = fixtures.convertToAbsoluteFromFixturesDir('javascriptinvoke/ReinitializeToDefault.via');
```

Comment:
This PR is not exercising the JavaScriptInvoke feature but is instead adding a new public api function.

This fixture should be placed in `test-it\karma\fixtures\publicapi\ReinitializeToDefault.via`